molecule/default/molecule.yml

```diff
@@ -21,6 +21,7 @@ provisioner:
         docker_rootful: false
         docker_user: debiantestinguser
         docker_user_bashrc: false
+        docker_expose_docker_api_via_tcp: true
```

Comment:
You just asked to extend it for `almalinux9` and `focalroot`. Not sure why only these, so I extended all the OS-types with the var.

Reply:
So the tests will test both options on different platforms

Reply:
Got you, 1min